Tests/Unit/SOGoTest.h

```diff
@@ -52,6 +54,9 @@
 
 - (BOOL) run;
 
+- (NSString*) diffForString: (NSString*) str1
+                  andString: (NSString*) str2;
```

Comment:
As we are creating a new `NSString`, according to Objective-C standard, we should start the message with string prefix, as developers must know this returned value is autoreleased and no further memory management action is required. So what about changing to `stringFromDiffBetween: str1 and: str2`?